nservicebus/scalability-and-ha/scale-out.md

```diff
@@ -72,4 +73,4 @@ snippet:Routing-FileBased-Broker
 
 In that case the sender will use round-robin distribution like in the MSMQ case.
 
-NOTE: Even when physical routing is specified (e.g. in form of the routing file) the transport is able to ignore it if it is not possible to implement a particular scenario. An example is RabbitMQ which delivers the events always to the shared queue. The reason for that is lack of built-in `round-robin` exchange type that could be used to bind event exchanges to individual queues.
\ No newline at end of file
+NOTE: Even when physical routing is specified (e.g. in form of the routing file) the transport is able to ignore it if it is not possible to implement a particular scenario. An example is RabbitMQ which delivers the events always to the shared queue. The reason for that is lack of built-in `round-robin` exchange type that could be used to bind event exchanges to individual queues.
```

Comment:
@SzymonPobiega How about:

However, it may not always be possible to route according to the physical routing file based on the transport that's being used. For example, the RabbitMQ transport always delivers the events to the shared queue regardless of the routing specified. This is because some transports lack built-in `round-robin` exchange type which could be used to bind event exchanges to individual queues.